tests/unit/test_domain_to_resolver.cairo

```diff
@@ -0,0 +1,16 @@
+%lang starknet
+from src.naming.utils import domain_to_resolver, write_domain_data, DomainData
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.alloc import alloc
+
+@external
+func test_domain_to_resolver{syscall_ptr: felt*, range_check_ptr, pedersen_ptr: HashBuiltin*}() {
+    write_domain_data(2, new ('charlie', 'delta'), DomainData(1, 'resolver', 123, -1, 0, 0));
+    let (resolver: felt, id: felt) = domain_to_resolver(
+        4, new ('alpha', 'bravo', 'charlie', 'delta'), 1
+    );
+    assert resolver = 'resolver';
+    assert id = 2;
```

Comment:
je comprend pas pq tu renvoies id et resolver + Peux tu rajouter des commentaires sous forme de `Should ...` pour expliquer